pkg/tfvars/aws/aws.go

```diff
@@ -24,7 +25,7 @@ type config struct {
 }
 
 // TFVars generates AWS-specific Terraform variables launching the cluster.
-func TFVars(masterConfigs []*v1beta1.AWSMachineProviderConfig, workerConfigs []*v1beta1.AWSMachineProviderConfig) ([]byte, error) {
+func TFVars(masterConfigs []*v1beta1.AWSMachineProviderConfig, workerConfigs []*v1beta1.AWSMachineProviderConfig, customEndpointsMap map[string]string) ([]byte, error) {
```

Comment:
same as https://github.com/openshift/installer/pull/2073/files#r306984378

Reply:
Removed the type from the variable name.